### PR TITLE
Add reloader to civo marketplace

### PR DESCRIPTION
**Pull request by dirien**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot)

# Install
image
image

# Running pods
image

# Uninstall
image

### CODE REVIEW

**Comment by dirien** [comment]
Hi, any updates on this 😅

File: reloader/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Reloader
```
Comment (saiyam1814) [inline reloader/manifest.yaml:2]: name: reloader
title: "Reloader"
version: "v0.0.91"

Can we have like this fo the manifest.yaml

File: reloader/post_install.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+## Reloader
```
Comment (saiyam1814) [inline reloader/post_install.md:1]: I think that is a long post_install 
can we have a smaller necessary version of this which is required to run and interact with the application ? 

```
A final optional file is the post_install.md which the user will be able to view in the web UI to see additional instructions on how to use the application. It's recommended to keep this short and succinct, if you need to manage LOTS of content, then please create a community learn guide and link to it from within the post_install.md
```

**Comment by saiyam1814** [comment]
> Hi, any updates on this 😅

Added a few comments :)

File: reloader/manifest.yaml (hunk 1 above)
Comment (dirien) [reply reloader/manifest.yaml:2]: ok, will change :)

File: reloader/post_install.md (hunk 2 above)
Comment (dirien) [reply reloader/post_install.md:1]: ok, i shorted it down. The link for more details is also there :)

**Comment by dirien** [comment]
hi @saiyam1814,

thanks for our feedback. I did all the changes. Thank you.

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**